src/js/components/Tip/Tip.js

```diff
@@ -63,7 +63,9 @@ const Tip = forwardRef(({ children, content, dropProps, plain }, tipRef) => {
         target={componentRef.current}
         trapFocus={false}
         key="tip-drop"
-        plain
+        background="none"
+        elevation="none"
+        margin="none"
```

Comment:
I can also add `round="0px"` but since it didn't have an impact on the UI, I preferred to keep the CSS clean. Let me know if you think it should be added.